Tasks/DeployVisualStudioTestAgent/VerifyTestMachinesAreInUse.ps1

```diff
@@ -0,0 +1,67 @@
+function Check-TestAgentIsRunning([string] $Version)
+{
+	$AvlVersion = Locate-TestVersion
+	if($AvlVersion)
+	{
+		$Version = $AvlVersion
+	}
+	Write-Verbose "VS Agent version $Version" -verbose
+	
+	$testAgentPath = "HKLM:\SOFTWARE\Microsoft\VisualStudio\{0}\EnterpriseTools\QualityTools\Agent" -f $Version
+	
+	if (-not (Test-Path $testAgentPath))
+	{
+		$testAgentPath = "HKLM:\SOFTWARE\Wow6432Node\Microsoft\VisualStudio\{0}\EnterpriseTools\QualityTools\Agent" -f $Version
+	}
+	
+	if (-not (Test-Path $testAgentPath))
+	{
+        Write-Verbose "Test Agent is not running as Path doesn't exist" -verbose
```

Comment:
user comments need to have a more understandable error and the corrective action.